src/main/scala/org/ergoplatform/local/ErgoStatsCollector.scala

```diff
@@ -155,6 +164,7 @@ object ErgoStatsCollector {
                       headersScore: Option[BigInt],
                       bestFullBlockOpt: Option[ErgoFullBlock],
                       fullBlocksScore: Option[BigInt],
+                      maxPeerHeight : Option[Int],
```

Comment:
Scaladoc would be good, this is one of the fields that is not self-explanatory.

Reply:
[69618c3](https://github.com/ergoplatform/ergo/pull/1683/commits/69618c3ecf757ba7877c95bf89b82f722774cfa7)
like this?